pkg/operator/ceph/cluster/controller.go

```diff
@@ -755,14 +756,22 @@ func (c *ClusterController) onDelete(obj interface{}) {
 }
 
 func (c *ClusterController) handleDelete(cluster *cephv1.CephCluster, retryInterval time.Duration) error {
-
+	flexDriverEnabled := os.Getenv(enableFlexDriver) == "true"
```

Comment:
Hum shouldn't this simply be `flexDriverEnabled := os.Getenv(enableFlexDriver)`, otherwise `flexDriverEnabled` will always be `true`, right?

Reply:
...unless I messed up. :D But I don't think so.

Reply:
My bad, you're right, still waking up I guess ^^